server/src/main/java/org/opensearch/rest/action/cat/RestNodesAction.java

```diff
@@ -310,6 +310,19 @@ protected Table getTableWithHeader(final RestRequest request) {
         );
         table.addCell("search.scroll_total", "alias:scto,searchScrollTotal;default:false;text-align:right;desc:completed scroll contexts");
 
+        table.addCell(
+            "search.point_in_time_current",
+            "alias:scc,searchPointInTimeCurrent;default:false;text-align:right;desc:open pit contexts"
+        );
+        table.addCell(
+            "search.point_in_time_time",
+            "alias:scti,searchPointInTimeTime;default:false;text-align:right;desc:time pit contexts held open"
```

Comment:
lets use term 'point in time' instead of 'pit' everywhere

Reply:
Updated